### ISSUE TITLE
views_with_side() should return indices of views in the order of the sides arg

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#263]
currently doesn't sort based on sides

**Comment by jordansread** [comment]
shows up here:

``` r
usrDef <- gsplot(mar=c(4,4,4,4), xaxs='r', yaxs='r') %>% 
    points(x=1, y=2, side=c(3,2), legend.name="Points 1", cex=3, xlab='cat') %>% 
    points(x=3:10,y=4:11, side=c(1,2))
xlim(usrDef, side = c(1,3))
# same output as 
xlim(usrDef, side = c(3,1))
```

**Comment by jordansread** [comment]
fixed in #319
